### ISSUE TITLE
Pokemon sightings / pokemon predictions

### ISSUE THREAD

**Issue by goldbergtatyana** [PokemonGoers/Catch-em-all#154]
Hey @PokemonGoers/catch-em-all , thanks for your creative idea of showing pokemons sightings and predictions using the two sliders. The implementation of sliders is beautiful!

Few comments:
- please add a mouse hover text above the icon for sightings and predictions (the blue circle with the white filter) telling something like "show pokemon sightings and predictions"
- we are making pokemon predictions for only the next 15 minutes. @PokemonGoers/predictpokemon-2 please correct me if the interval of 15 minutes is actually larger? So, the many options provided by the slider do unfortunately not apply for us... Instead a simple switch for showing predictions could be sufficient.

**Comment by johartl** [comment]
> please add a mouse hover text

You are aware that hover texts won't work on touch devices? Maybe we should find a solution which works on all devices consistently.

**Comment by bensLine** [comment]
Predictions are for the current timestamp. every 15min the training set for the classifier is updated

**Comment by goldbergtatyana** [comment]
Nope, not aware :) Well, the icon needs to be self explanatory somehow. What if we have a picture of a pokemon, or even better yet a pulsating image of the location pin (so that it immediately grabs the attention of the user) instead of the blue circle?

**Comment by gyachdav** [comment]
You guys know that currently we do not get updated sightings since all sources are down (except for twitter but the number of reported sightings there is very low). Please verify with @sacdallago and @PokemonGoers/pokedata but currently there is no point in trying to update the model.

**Comment by MajorBreakfast** [comment]
@goldbergtatyana 

> Instead a simple switch for showing predictions could be sufficient

No problem

> the icon needs to be self explanatory somehow

We could add a label "Filter Options". We'll come up with something

**Comment by sacdallago** [comment]
@gyachdav I doubt that the services will be up anytime soon! We have to drive people to use twitter in a way that works for our app.
